src/ResourceManager/Media/Commands.Media/Commands.Media.csproj

```diff
@@ -0,0 +1,154 @@
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="Microsoft.Azure.Management.Media">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Media.0.1.0\lib\net45\Microsoft.Azure.Management.Media.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Net" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Runtime.Serialization" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Common\AzureMediaServiceCmdletBase.cs" />
+    <Compile Include="MediaService\GetAzureRmMediaServiceNameAvailability.cs" />
+    <Compile Include="MediaService\RemoveAzureRmMediaService.cs" />
+    <Compile Include="MediaService\NewAzureRmMediaService.cs" />
+    <Compile Include="MediaService\GetAzureRmMediaService.cs" />
+    <Compile Include="MediaService\SetAzureRmMediaService.cs" />
+    <Compile Include="Models\ModelExtensions.cs" />
+    <Compile Include="Models\PSApiEndpoint.cs" />
+    <Compile Include="Models\PSCheckNameAvailabilityOutput.cs" />
+    <Compile Include="Models\PSMediaService.cs" />
+    <Compile Include="Models\PSServiceKey.cs" />
+    <Compile Include="Models\PSServiceKeys.cs" />
+    <Compile Include="Models\PSMediaServiceProperties.cs" />
+    <Compile Include="Models\PSStorageAccount.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="ServiceKey\SyncAzureRmMediaServiceStorageKeys.cs" />
+    <Compile Include="ServiceKey\SetAzureRmMediaServiceKey.cs" />
+    <Compile Include="ServiceKey\GetAzureRmMediaServiceKeys.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Microsoft.Azure.Commands.Media.dll-help.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <SubType>Designer</SubType>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
+      <Project>{d3804b64-c0d3-48f8-82ec-1f632f833c9e}</Project>
+      <Name>Commands.Common.Authentication</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Tags\Commands.Tags\Commands.Tags.csproj">
```

Comment:
References to Commands.Tags are not allowed.  The common code for tag management is in Commands.ResourceManager.Common